src/main.js

```diff
@@ -3,13 +3,23 @@ import InfoView from './view/main-info-view.js';
 import TripFilter from './view/trip-filters.js';
 import FormSort from './view/sort-main.js';
 import EventPresenter from './presenter/trip-event-presenter.js';
+import CreatePointList from './service/service.js';
+import ModelPoint from './model/point-model.js';
+import ModelOffer from './model/offer-model.js';
+import ModelDestinatios from './model/destination-model.js';
 
-const eventPresenter = new EventPresenter({listContainer: document.querySelector('.trip-events')});
+
+const mockService = new CreatePointList();
+const mockPoint = new ModelPoint(mockService);
```

Comment:
эти модели можно не называть моковыми. Ты будешь всегда работать с ними, какие в них данные определяет модель внутри, т.е. для приложения это уже твоя модель данных, то какие сейчас в ней данные моковые или не моковые значения не имеет
```suggestion
const pointsModel = new ModelPoint(mockService);
const offersModel = new ModelOffer(mockService);
const destinationsModel = new ModelDestinatios(mockService);
```